docs/docs/usage.md

```diff
@@ -48,7 +48,23 @@ Connect to recreate the environment and re-run the notebook at a later time.
 publish an HTML snapshot of the notebook to RStudio Connect. HTML snapshots are static and 
 cannot be scheduled or re-run on the RStudio Connect server.
 
-<img class="border" src="../images/manage.png" class="block" alt="publish dialog">
+<img class="border" src="../images/rsconnect-jupyter-usage.png" class="block" alt="publish dialog">
+
+### Hide Input
+
+There are two options for hiding input code cells in published Jupyter Notebooks. You can choose to hide all the input code cells, or only hide the cells that have been explicitly tagged. 
```

Comment:
Can we do something similar to the user guide entry?
There are two options for hiding input code cells in Jupyter Notebooks published
to RStudio Connect:

- Hide all input code cells
- Hide only selected input code cells

Reply:
yes